pkg/config/env.go

```diff
@@ -184,6 +184,10 @@ var Config = struct {
 	// "HS256" and "RS256" supported
 	JWTAuthSigningMethod string `env:"FLAGR_JWT_AUTH_SIGNING_METHOD" envDefault:"HS256"`
 
+	// auth proxy in front of flagr to identify user
+	AuthProxyEnabled           bool     `env:"FLAGR_AUTH_PROXY_ENABLED" envDefault:"false"`
```

Comment:
How about calling it `HeaderAuth`, compatible with `JWTAuth` above. 

```
HeaderAuthEnabled bool
HeaderAuthUserField string
```

Reply:
Thanks,  sounds better